### PR TITLE
Create issue and pr templates

### PR DESCRIPTION
**Pull request by DavidKnott**
Creates issue and pr templates in an attempt to make issues and pull requests easier to understand.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+* viper Version: x.x.x
+* pyethereum Version: x.x.x
+* OS: osx/linux/win
+* Python Version (python --version):
+* Environment (output of `pip freeze`):
+
+### What is wrong?
```
Comment (yograterol) [inline .github/ISSUE_TEMPLATE.md]: May be a issue could not be a trouble, what do you think?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+### What was wrong?
+
```
Comment (yograterol) [inline .github/PULL_REQUEST_TEMPLATE.md]: Same, a PR could be a feature ;)

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (DavidKnott) [reply .github/ISSUE_TEMPLATE.md]: I see your point, what do you think of `What's your issue about?`

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (DavidKnott) [reply .github/PULL_REQUEST_TEMPLATE.md]: And `What's your pull request in response to?`

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (yograterol) [reply .github/ISSUE_TEMPLATE.md]: Yes, it's better

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (yograterol) [reply .github/PULL_REQUEST_TEMPLATE.md]: We could get the questions from moby repo:


**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**Comment by fubuloubu** [comment]
Is there a way to see these in action? I am not familiar with the templates?

I mean honestly, what's the harm with pulling these in (the templates at least) and seeing how they look in real life?

**Comment by DavidKnott** [comment]
I'm not sure how to see them in action without pulling them in, I'll make the changes requested and then they can be pulled in.

**Comment by yograterol** [comment]
@fubuloubu you can copy and paste the content here, sounds obvious but it's the same behavior, by example:

# Issue template

* viper Version: x.x.x
* pyethereum Version: x.x.x
* OS: osx/linux/win
* Python Version (python --version):
* Environment (output of `pip freeze`):

### What is wrong?

Please include information like:

* full output of the error you received
* what command you ran
* the code that caused the failure (see [this link](https://help.github.com/articles/basic-writing-and-formatting-syntax/) for help with formatting code)

### How can it be fixed

Fill this in if you know how to fix it.

#### Cute Animal Picture

> put a cute animal picture here.

**Comment by yograterol** [comment]
the contributor should fill the template replacing the default content.

**Comment by fubuloubu** [comment]
Also, @DavidKnott you should start doing bumbversion every so often now that you're working full time on it.

**Pull request merged**